### PR TITLE
Newsletter 212: Translate to German

### PR DESCRIPTION
**Pull request by tschweg**
*default minimum transaction relay feerate* has been translated to *Standardminimumgebührenrate für Transaktionsweiterleitungen*, which imho is a bit convoluted and might confuse native German speakers. Same goes for *opt-in* / *standardmäßigen Einwilligung (opt-in)*. I wasn't able to come up with a more concise translation. 

There's probably a story for creating a small reference sheet with terms that should be used in their English native form.

### CODE REVIEW

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+    Die Standardminimumgebührenrate zu senken wurde bereits in der
+    Vergangenheit diskutiert (siehe [Newsletter #3][news3 min]), wurde aber
+    [nicht in Bitcoin Core übernommen][bitcoin core #13922].
+    Die [Diskussion um das Thema][chauhan min] erwachte in den letzten Wochen
+    wieder:
+
+    - *Wirksamkeit individueller Änderungen:* Es wurde von mehreren Personen
+      [debattiert][todd min] wie effektiv die Änderung der Richtlinien für
+      einzelne Node-Betreiber tatsächlich war.
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:37]:
```suggestion
    - *Wirksamkeit individueller Änderungen:* Es wurde von mehreren Personen
      [debattiert][todd min] wie effektiv es für einzelne Node-Betreiber war, die 
      Richtlinien ihres eigenen Nodes zu ändern.
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+    - *Vergangene Fehlschläge:* Es wurde [erwähnt][harding min], dass der
+      vorherige Versuch, die Standardgebührenrate zu senken dadurch behindert
+      wurde, dass die niedrigere Rate auch die Kosten für mehrere kleinere
+      Denial-of-Service (DoS) Angriffe senkte.
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:42]:
```suggestion
      wurde, dass niedrigere Raten auch die Kosten für mehrere untergeordnete
      Denial-of-Service (DoS) Angriffe senken würden.
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+    - *Alternative Weiterleitungskriterien:*
+      Es wurde [vorgeschlagen][todd min2], dass Transaktionen, die bestimmte
+      Standardkriterien verletzen (z. B. die Standard-Minimumgebührenrate)
+      stattdessen einige separate Kriterien erfüllen könnten, die DoS-Angriffe
+      kostspielig machen würden. Zum Beispiel, wenn eine bescheidene Menge von
+      Hashcash-ähnlichen Arbeitsnachweisen zur Transaktionsweiterleitung
+      beitragen würde.
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:50]:
```suggestion
      stattdessen separate Kriterien erfüllen könnten, die DoS-Angriffe
      kostspielig machen würden. Zum Beispiel, wenn eine bescheidene Menge von
      Hashcash-ähnlichen Arbeitsnachweisen zur Transaktionsweiterleitung
      beitragen würde.
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+handhabt, als Bitcoin Core-spezifische Funktionalität in `src/node`."
+  a0link="https://bitcoincore.reviews/25527#l-35"
+
+  q1="Was sind die Validierungszwischenspeicher? Warum gehören sie in
+`src/kernel` gegenüber `src/node`?"
+  a1="Wenn ein neuer Block empfangen wird, ist der rechenintensivste Teil der
+Validierung die Skriptprüfung (d.h. die Prüfung der Signatur) der Transaktionen
+des Blocks. Da Knoten, die einen Mempool führen, diese Transaktionen in der
+Regel bereits gesehen und validiert haben, wird die Leistung der
+Blockvalidierung durch das Zwischenspeichern (erfolgreicher) Skript- und
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:84]:
```suggestion
Regel bereits gesehen und validiert haben, wird die Durchsatz der
Blockvalidierung durch das Zwischenspeichern (erfolgreicher) Skript- und
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25610][] bringt einen standardmäßigen Einwilligung (opt-in)
+  von RPCs und `-walletrbf` zu [RBF][topic rbf]. Diese folgt auf die in
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:144]: Der Absatz wurde nochmal nachgebessert nach Veröffentlichung des Newsletters:

```suggestion
- [Bitcoin Core #25610][] aktiviert standardmäßig die `-walletrbf` Startoption und führt außerdem dazu, dass die RPCs  `createrawtransaction` und `createpsbt` standardmäßig ersetzbare Transaktionen erzeugen. Transaktionen die mit der grafischen Benutzeroberfläche erzeugt worden waren, verwendeten schon zuvor standardmäßig [eingewilligte Ersetzbarkeit (opt-in RBF)][topic rbf]. Dies folgt auf die in
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+  [Newsletter #208][news208 core RBF] erwähnte Aktualisierung, die es Node-
+  Betreibern ermöglicht das Transaktionsersetzungsverhalten ihres Nodes von der
+  Standard-RBF (BIP125) auf Voll-RBF umzustellen. Standardmäßige RPC
+  Einwilligung wurde im Jahr 2017 in [Bitcoin Core #9527][] vorgeschlagen.
+  Die damaligen Haupteinwände waren: Neuartigkeit der Funktion,
+  mangelnde Funktionalität zum Erhöhen von Transaktionen und fehlende
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:150]:
```suggestion
  mangelnde Funktionalität zum Erhöhen von Transaktionsgebühren und fehlende
```

File: _posts/de/newsletters/2022-08-10-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+  Funktionalität der GUI RBF zu deaktivieren; welche seitdem alle behoben
+  wurden.
+
+- [Bitcoin Core #24584][] ändert die [Coin-Auswahl][topic coin selection], um
+  Eingabe-Sets zu bevorzugen die aus einem einzigen Ausgabetyp bestehen.
+  Dies adressiert Szenarien, in denen gemischte Eingabe-Sets die
+  Wechselgeldausgabe vorangegangener Transaktionen enthüllen, und folgt auf
+  eine damit verbundene Verbesserung des Datenschutzes, die immer die
+  [Änderungsart mit einer Empfängerausgabe abgleicht][#23789].
```
Comment (Xekyo) [inline _posts/de/newsletters/2022-08-10-newsletter.md:159]:
```suggestion
  eine damit verbundene Verbesserung des Datenschutzes, die immer den
  [Ausgabetyp des Wechselgelds mit einer Empfängerausgabe abgleicht][#23789].
```

**Review by Xekyo — COMMENTED** [review]
Sorry for the late review, I missed this PR yesterday unfortunately. I think your translations for the feerate and opt-in RBF sound fine, I don't have a better idea spontaneously either. I have a few suggestions for alternative expressions, and in a couple instances, I think other formulations would better capture the original intent.

As usual, please feel free to take and leave any suggestions. Thanks for your work

**Review by bitschmidty — COMMENTED** [review]
ACK. Will let @Xekyo provide a final review on the content.

**Review by Xekyo — APPROVED** [review]
ACK, looks good to me. In the future we might want to start adding a list of standard terms for German as well. I noticed that we write “keypath” and “Skriptpfade”, but it's not keeping me awake at night. :)

**Pull request merged**